src/cpp/cuda.hpp

```diff
@@ -997,8 +997,8 @@ namespace pycuda
       CUstream m_stream;
 
     public:
-      stream(unsigned int flags=0)
-      { CUDAPP_CALL_GUARDED(cuStreamCreate, (&m_stream, flags)); }
```

Comment:
Do you have a sense when this was added to CUDA? If it's sufficiently recent, its use should be covered by `#ifdef`s, with error messages if the feature is not supported.

Reply:
Well, let's start at the beginning. When (i.e. at what version) was this added to CUDA?

Reply:
I can see that it was in 8.0 at least in https://docs.nvidia.com/cuda/archive/8.0/cuda-driver-api/group__CUDA__CTX.html but I'm having trouble finding a list of variables for older versions for the moment.

Reply:
I found the variable name in 7.5 release notes, but it implied that it already existed earlier.